### ISSUE TITLE
Backend don't start due to python3 core dumped

### ISSUE THREAD

**Issue by fpoisson35** [LibrePhotos/librephotos#1097]
Hello,

I have updated to latest librephotos-docker instance and now my backend doesn't start due to a core dump from python. When i open a shell on the docker container and run this i have a core dump exception :
root@ff291daac0da:/code# python3 -c "import torch"
Illegal instruction (core dumped)
```
root@ff291daac0da:/code# python3 -v -c "import torch"
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'posix' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# /usr/lib/python3.11/encodings/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/encodings/__init__.py
# code object from '/usr/lib/python3.11/encodings/__pycache__/__init__.cpython-311.pyc'
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <class '_frozen_importlib.FrozenImporter'>
# /usr/lib/python3.11/encodings/__pycache__/aliases.cpython-311.pyc matches /usr/lib/python3.11/encodings/aliases.py
# code object from '/usr/lib/python3.11/encodings/__pycache__/aliases.cpython-311.pyc'
import 'encodings.aliases' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d41db50>
import 'encodings' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d409f90>
# /usr/lib/python3.11/encodings/__pycache__/utf_8.cpython-311.pyc matches /usr/lib/python3.11/encodings/utf_8.py
# code object from '/usr/lib/python3.11/encodings/__pycache__/utf_8.cpython-311.pyc'
import 'encodings.utf_8' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d40b950>
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <class '_frozen_importlib.FrozenImporter'>
import 'io' # <class '_frozen_importlib.FrozenImporter'>
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <class '_frozen_importlib.FrozenImporter'>
import '_collections_abc' # <class '_frozen_importlib.FrozenImporter'>
import 'genericpath' # <class '_frozen_importlib.FrozenImporter'>
import 'posixpath' # <class '_frozen_importlib.FrozenImporter'>
import 'os' # <class '_frozen_importlib.FrozenImporter'>
import '_sitebuiltins' # <class '_frozen_importlib.FrozenImporter'>
Processing user site-packages
Processing global site-packages
Adding directory: '/usr/local/lib/python3.11/dist-packages'
Processing .pth file: '/usr/local/lib/python3.11/dist-packages/coloredlogs.pth'
Processing .pth file: '/usr/local/lib/python3.11/dist-packages/easy-install.pth'
Processing .pth file: '/usr/local/lib/python3.11/dist-packages/zope.event-5.0-py3.11-nspkg.pth'
# /usr/lib/python3.11/__pycache__/types.cpython-311.pyc matches /usr/lib/python3.11/types.py
# code object from '/usr/lib/python3.11/__pycache__/types.cpython-311.pyc'
import 'types' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1e8410>
# /usr/lib/python3.11/importlib/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/importlib/__init__.py
# code object from '/usr/lib/python3.11/importlib/__pycache__/__init__.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/warnings.cpython-311.pyc matches /usr/lib/python3.11/warnings.py
# code object from '/usr/lib/python3.11/__pycache__/warnings.cpython-311.pyc'
import 'warnings' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1e9fd0>
import 'importlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1e9ad0>
# /usr/lib/python3.11/importlib/__pycache__/_abc.cpython-311.pyc matches /usr/lib/python3.11/importlib/_abc.py
# code object from '/usr/lib/python3.11/importlib/__pycache__/_abc.cpython-311.pyc'
import 'importlib._abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d208250>
# /usr/lib/python3.11/__pycache__/contextlib.cpython-311.pyc matches /usr/lib/python3.11/contextlib.py
# code object from '/usr/lib/python3.11/__pycache__/contextlib.cpython-311.pyc'
# /usr/lib/python3.11/collections/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/collections/__init__.py
# code object from '/usr/lib/python3.11/collections/__pycache__/__init__.cpython-311.pyc'
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/lib/python3.11/__pycache__/keyword.cpython-311.pyc matches /usr/lib/python3.11/keyword.py
# code object from '/usr/lib/python3.11/__pycache__/keyword.cpython-311.pyc'
import 'keyword' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d229c10>
# /usr/lib/python3.11/__pycache__/operator.cpython-311.pyc matches /usr/lib/python3.11/operator.py
# code object from '/usr/lib/python3.11/__pycache__/operator.cpython-311.pyc'
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d22a710>
# /usr/lib/python3.11/__pycache__/reprlib.cpython-311.pyc matches /usr/lib/python3.11/reprlib.py
# code object from '/usr/lib/python3.11/__pycache__/reprlib.cpython-311.pyc'
import 'reprlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d22b3d0>
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d208bd0>
# /usr/lib/python3.11/__pycache__/functools.cpython-311.pyc matches /usr/lib/python3.11/functools.py
# code object from '/usr/lib/python3.11/__pycache__/functools.cpython-311.pyc'
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d2103d0>
import 'contextlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d208390>
import 'importlib.util' # <class '_frozen_importlib.FrozenImporter'>
import 'importlib.machinery' # <class '_frozen_importlib.FrozenImporter'>
# possible namespace for /usr/local/lib/python3.11/dist-packages/zope
Processing .pth file: '/usr/local/lib/python3.11/dist-packages/zope.interface-6.1-py3.11-nspkg.pth'
# possible namespace for /usr/local/lib/python3.11/dist-packages/zope
# destroy zope
Adding directory: '/usr/lib/python3/dist-packages'
Processing .pth file: '/usr/lib/python3/dist-packages/distutils-precedence.pth'
# /usr/lib/python3/dist-packages/_distutils_hack/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3/dist-packages/_distutils_hack/__init__.py
# code object from '/usr/lib/python3/dist-packages/_distutils_hack/__pycache__/__init__.cpython-311.pyc'
import '_distutils_hack' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d262090>
# /usr/lib/python3.11/__pycache__/sitecustomize.cpython-311.pyc matches /usr/lib/python3.11/sitecustomize.py
# code object from '/usr/lib/python3.11/__pycache__/sitecustomize.cpython-311.pyc'
import 'sitecustomize' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d263650>
import 'site' # <class '_frozen_importlib.FrozenImporter'>
Python 3.11.4 (main, Jun  9 2023, 07:59:55) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
# /usr/local/lib/python3.11/dist-packages/torch/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/dist-packages/torch/__init__.py
# code object from '/usr/local/lib/python3.11/dist-packages/torch/__pycache__/__init__.cpython-311.pyc'
import 'math' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/lib/python3.11/__pycache__/platform.cpython-311.pyc matches /usr/lib/python3.11/platform.py
# code object from '/usr/lib/python3.11/__pycache__/platform.cpython-311.pyc'
# /usr/lib/python3.11/re/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/re/__init__.py
# code object from '/usr/lib/python3.11/re/__pycache__/__init__.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/enum.cpython-311.pyc matches /usr/lib/python3.11/enum.py
# code object from '/usr/lib/python3.11/__pycache__/enum.cpython-311.pyc'
import 'enum' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d291cd0>
# /usr/lib/python3.11/re/__pycache__/_compiler.cpython-311.pyc matches /usr/lib/python3.11/re/_compiler.py
# code object from '/usr/lib/python3.11/re/__pycache__/_compiler.cpython-311.pyc'
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/lib/python3.11/re/__pycache__/_parser.cpython-311.pyc matches /usr/lib/python3.11/re/_parser.py
# code object from '/usr/lib/python3.11/re/__pycache__/_parser.cpython-311.pyc'
# /usr/lib/python3.11/re/__pycache__/_constants.cpython-311.pyc matches /usr/lib/python3.11/re/_constants.py
# code object from '/usr/lib/python3.11/re/__pycache__/_constants.cpython-311.pyc'
import 're._constants' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d2c4190>
import 're._parser' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d2b91d0>
# /usr/lib/python3.11/re/__pycache__/_casefix.cpython-311.pyc matches /usr/lib/python3.11/re/_casefix.py
# code object from '/usr/lib/python3.11/re/__pycache__/_casefix.cpython-311.pyc'
import 're._casefix' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d2c79d0>
import 're._compiler' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d2a9cd0>
# /usr/lib/python3.11/__pycache__/copyreg.cpython-311.pyc matches /usr/lib/python3.11/copyreg.py
# code object from '/usr/lib/python3.11/__pycache__/copyreg.cpython-311.pyc'
import 'copyreg' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d0d0d50>
import 're' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d290b10>
import 'platform' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d288350>
# /usr/lib/python3.11/__pycache__/textwrap.cpython-311.pyc matches /usr/lib/python3.11/textwrap.py
# code object from '/usr/lib/python3.11/__pycache__/textwrap.cpython-311.pyc'
import 'textwrap' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d290790>
# /usr/lib/python3.11/ctypes/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/ctypes/__init__.py
# code object from '/usr/lib/python3.11/ctypes/__pycache__/__init__.cpython-311.pyc'
# extension module '_ctypes' loaded from '/usr/lib/python3.11/lib-dynload/_ctypes.cpython-311-x86_64-linux-gnu.so'
# extension module '_ctypes' executed from '/usr/lib/python3.11/lib-dynload/_ctypes.cpython-311-x86_64-linux-gnu.so'
import '_ctypes' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f521d0d73d0>
# /usr/lib/python3.11/__pycache__/struct.cpython-311.pyc matches /usr/lib/python3.11/struct.py
# code object from '/usr/lib/python3.11/__pycache__/struct.cpython-311.pyc'
import '_struct' # <class '_frozen_importlib.BuiltinImporter'>
import 'struct' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d0e6250>
# /usr/lib/python3.11/ctypes/__pycache__/_endian.cpython-311.pyc matches /usr/lib/python3.11/ctypes/_endian.py
# code object from '/usr/lib/python3.11/ctypes/__pycache__/_endian.cpython-311.pyc'
import 'ctypes._endian' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d0f8490>
import 'ctypes' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d0d3790>
# /usr/lib/python3.11/__pycache__/inspect.cpython-311.pyc matches /usr/lib/python3.11/inspect.py
# code object from '/usr/lib/python3.11/__pycache__/inspect.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/ast.cpython-311.pyc matches /usr/lib/python3.11/ast.py
# code object from '/usr/lib/python3.11/__pycache__/ast.cpython-311.pyc'
import '_ast' # <class '_frozen_importlib.BuiltinImporter'>
import 'ast' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d10ea50>
# /usr/lib/python3.11/__pycache__/dis.cpython-311.pyc matches /usr/lib/python3.11/dis.py
# code object from '/usr/lib/python3.11/__pycache__/dis.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/opcode.cpython-311.pyc matches /usr/lib/python3.11/opcode.py
# code object from '/usr/lib/python3.11/__pycache__/opcode.cpython-311.pyc'
import '_opcode' # <class '_frozen_importlib.BuiltinImporter'>
import 'opcode' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d18c690>
import 'dis' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d129dd0>
# /usr/lib/python3.11/collections/__pycache__/abc.cpython-311.pyc matches /usr/lib/python3.11/collections/abc.py
# code object from '/usr/lib/python3.11/collections/__pycache__/abc.cpython-311.pyc'
import 'collections.abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d17dad0>
# /usr/lib/python3.11/__pycache__/linecache.cpython-311.pyc matches /usr/lib/python3.11/linecache.py
# code object from '/usr/lib/python3.11/__pycache__/linecache.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/tokenize.cpython-311.pyc matches /usr/lib/python3.11/tokenize.py
# code object from '/usr/lib/python3.11/__pycache__/tokenize.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/token.cpython-311.pyc matches /usr/lib/python3.11/token.py
# code object from '/usr/lib/python3.11/__pycache__/token.cpython-311.pyc'
import 'token' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1a20d0>
import 'tokenize' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d194d10>
import 'linecache' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d17d9d0>
import 'inspect' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d0d6250>
# /usr/local/lib/python3.11/dist-packages/torch/__pycache__/_utils.cpython-311.pyc matches /usr/local/lib/python3.11/dist-packages/torch/_utils.py
# code object from '/usr/local/lib/python3.11/dist-packages/torch/__pycache__/_utils.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/traceback.cpython-311.pyc matches /usr/lib/python3.11/traceback.py
# code object from '/usr/lib/python3.11/__pycache__/traceback.cpython-311.pyc'
import 'traceback' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1c5990>
# /usr/lib/python3.11/__pycache__/typing.cpython-311.pyc matches /usr/lib/python3.11/typing.py
# code object from '/usr/lib/python3.11/__pycache__/typing.cpython-311.pyc'
# extension module '_typing' loaded from '/usr/lib/python3.11/lib-dynload/_typing.cpython-311-x86_64-linux-gnu.so'
# extension module '_typing' executed from '/usr/lib/python3.11/lib-dynload/_typing.cpython-311-x86_64-linux-gnu.so'
import '_typing' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f521cf4e050>
import 'typing' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cf41c90>
import 'torch._utils' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d1bac90>
# /usr/local/lib/python3.11/dist-packages/torch/__pycache__/_utils_internal.cpython-311.pyc matches /usr/local/lib/python3.11/dist-packages/torch/_utils_internal.py
# code object from '/usr/local/lib/python3.11/dist-packages/torch/__pycache__/_utils_internal.cpython-311.pyc'
# /usr/lib/python3.11/logging/__pycache__/__init__.cpython-311.pyc matches /usr/lib/python3.11/logging/__init__.py
# code object from '/usr/lib/python3.11/logging/__pycache__/__init__.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/weakref.cpython-311.pyc matches /usr/lib/python3.11/weakref.py
# code object from '/usr/lib/python3.11/__pycache__/weakref.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/_weakrefset.cpython-311.pyc matches /usr/lib/python3.11/_weakrefset.py
# code object from '/usr/lib/python3.11/__pycache__/_weakrefset.cpython-311.pyc'
import '_weakrefset' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfcd650>
import 'weakref' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfbee10>
# /usr/lib/python3.11/__pycache__/string.cpython-311.pyc matches /usr/lib/python3.11/string.py
# code object from '/usr/lib/python3.11/__pycache__/string.cpython-311.pyc'
import '_string' # <class '_frozen_importlib.BuiltinImporter'>
import 'string' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfcf8d0>
# /usr/lib/python3.11/__pycache__/threading.cpython-311.pyc matches /usr/lib/python3.11/threading.py
# code object from '/usr/lib/python3.11/__pycache__/threading.cpython-311.pyc'
import 'threading' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfcfad0>
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>
import 'logging' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfac850>
# /usr/lib/python3.11/__pycache__/tempfile.cpython-311.pyc matches /usr/lib/python3.11/tempfile.py
# code object from '/usr/lib/python3.11/__pycache__/tempfile.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/shutil.cpython-311.pyc matches /usr/lib/python3.11/shutil.py
# code object from '/usr/lib/python3.11/__pycache__/shutil.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/fnmatch.cpython-311.pyc matches /usr/lib/python3.11/fnmatch.py
# code object from '/usr/lib/python3.11/__pycache__/fnmatch.cpython-311.pyc'
import 'fnmatch' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d026c50>
import 'errno' # <class '_frozen_importlib.BuiltinImporter'>
import 'zlib' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/lib/python3.11/__pycache__/bz2.cpython-311.pyc matches /usr/lib/python3.11/bz2.py
# code object from '/usr/lib/python3.11/__pycache__/bz2.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/_compression.cpython-311.pyc matches /usr/lib/python3.11/_compression.py
# code object from '/usr/lib/python3.11/__pycache__/_compression.cpython-311.pyc'
import '_compression' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d037610>
# extension module '_bz2' loaded from '/usr/lib/python3.11/lib-dynload/_bz2.cpython-311-x86_64-linux-gnu.so'
# extension module '_bz2' executed from '/usr/lib/python3.11/lib-dynload/_bz2.cpython-311-x86_64-linux-gnu.so'
import '_bz2' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f521ce405d0>
import 'bz2' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d036450>
# /usr/lib/python3.11/__pycache__/lzma.cpython-311.pyc matches /usr/lib/python3.11/lzma.py
# code object from '/usr/lib/python3.11/__pycache__/lzma.cpython-311.pyc'
# extension module '_lzma' loaded from '/usr/lib/python3.11/lib-dynload/_lzma.cpython-311-x86_64-linux-gnu.so'
# extension module '_lzma' executed from '/usr/lib/python3.11/lib-dynload/_lzma.cpython-311-x86_64-linux-gnu.so'
import '_lzma' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f521ce41b50>
import 'lzma' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521ce40b50>
import 'shutil' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d00ef50>
# /usr/lib/python3.11/__pycache__/random.cpython-311.pyc matches /usr/lib/python3.11/random.py
# code object from '/usr/lib/python3.11/__pycache__/random.cpython-311.pyc'
# /usr/lib/python3.11/__pycache__/bisect.cpython-311.pyc matches /usr/lib/python3.11/bisect.py
# code object from '/usr/lib/python3.11/__pycache__/bisect.cpython-311.pyc'
import '_bisect' # <class '_frozen_importlib.BuiltinImporter'>
import 'bisect' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521ce612d0>
import '_random' # <class '_frozen_importlib.BuiltinImporter'>
import '_sha512' # <class '_frozen_importlib.BuiltinImporter'>
import 'random' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521d00f710>
import 'tempfile' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfbf5d0>
import 'torch._utils_internal' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521cfac210>
# /usr/local/lib/python3.11/dist-packages/torch/__pycache__/torch_version.cpython-311.pyc matches /usr/local/lib/python3.11/dist-packages/torch/torch_version.py
# code object from '/usr/local/lib/python3.11/dist-packages/torch/__pycache__/torch_version.cpython-311.pyc'
# /usr/local/lib/python3.11/dist-packages/torch/__pycache__/version.cpython-311.pyc matches /usr/local/lib/python3.11/dist-packages/torch/version.py
# code object from '/usr/local/lib/python3.11/dist-packages/torch/__pycache__/version.cpython-311.pyc'
import 'torch.version' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521ce63610>
import 'torch.torch_version' # <_frozen_importlib_external.SourceFileLoader object at 0x7f521ce62f10>
Illegal instruction (core dumped)
root@ff291daac0da:/code# pip3 show torch
DEPRECATION: Loading egg at /usr/local/lib/python3.11/dist-packages/dlib-19.24.99-py3.11-linux-x86_64.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330
Name: torch
Version: 2.1.0+cpu
Summary: Tensors and Dynamic neural networks in Python with strong GPU acceleration
Home-page: https://pytorch.org/
Author: PyTorch Team
Author-email: packages@pytorch.org
License: BSD-3
Location: /usr/local/lib/python3.11/dist-packages
Requires: filelock, fsspec, jinja2, networkx, sympy, typing-extensions
Required-by: sentence-transformers, timm, torchvision
root@ff291daac0da:/code# 
```

And so the backend do not start the gunicorn process for the same reason : 
```  ..
[2023-12-16 12:04:18 +0000] [83] [ERROR] Worker (pid:432) was sent code 132!
[2023-12-16 12:04:18 +0000] [438] [INFO] Booting worker with pid: 438
[2023-12-16 12:04:19 +0000] [83] [ERROR] Worker (pid:433) was sent code 132!
[2023-12-16 12:04:19 +0000] [439] [INFO] Booting worker with pid: 439
use SECRET_KEY from file
use SECRET_KEY from file
Fatal Python error: Illegal instruction
Current thread 0x00007ff92ff54040 (most recent call first):
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 1233 in create_module
  File "<frozen importlib._bootstrap>", line 573 in module_from_spec
  File "<frozen importlib._bootstrap>", line 676 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/usr/local/lib/python3.11/dist-packages/torch/__init__.py", line 235 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/im2txt/sample.py", line 5 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/photo.py", line 22 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/person.py", line 8 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/album_auto.py", line 6 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/__init__.py", line 1 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "<frozen importlib._bootstrap>", line 1204 in _gcd_import
  File "/usr/lib/python3.11/importlib/__init__.py", line 126 in import_module
  File "/usr/local/lib/python3.11/dist-packages/django/apps/config.py", line 269 in import_models
  File "/usr/local/lib/python3.11/dist-packages/django/apps/registry.py", line 116 in populate
  File "/usr/local/lib/python3.11/dist-packages/django/__init__.py", line 24 in setup
  File "/usr/local/lib/python3.11/dist-packages/django/core/wsgi.py", line 12 in get_wsgi_application
  File "/code/librephotos/wsgi.py", line 11 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "<frozen importlib._bootstrap>", line 1204 in _gcd_import
  File "/usr/lib/python3.11/importlib/__init__.py", line 126 in import_module
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/util.py", line 371 in import_app
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 48 in load_wsgiapp
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 58 in load
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 67 in wsgi
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 146 in load_wsgi
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 134 in init_process
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/ggevent.py", line 147 in init_process
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 609 in spawn_worker
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 642 in spawn_workers
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 571 in manage_workers
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 211 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 72 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 236 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 67 in run
  File "/usr/local/bin/gunicorn", line 8 in <module>
Extension modules: greenlet._greenlet, zope.interface._zope_interface_coptimizations, gevent.libev.corecext, gevent._gevent_c_greenlet_primitives, gevent._gevent_c_hub_local, gevent._gevent_c_waiter, gevent._gevent_c_hub_primitives, gevent._gevent_c_ident, gevent._gevent_cgreenlet, gevent._gevent_c_abstract_linkable, gevent._gevent_cevent, gevent._gevent_cqueue, gevent._gevent_c_semaphore, gevent._gevent_c_imap, gevent._gevent_clocal, psutil._psutil_linux, psutil._psutil_posix, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator, markupsafe._speedups, charset_normalizer.md (total: 32)
Fatal Python error: Illegal instruction
Current thread 0x00007ff92ff54040 (most recent call first):
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 1233 in create_module
  File "<frozen importlib._bootstrap>", line 573 in module_from_spec
  File "<frozen importlib._bootstrap>", line 676 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/usr/local/lib/python3.11/dist-packages/torch/__init__.py", line 235 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/im2txt/sample.py", line 5 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/photo.py", line 22 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/person.py", line 8 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/album_auto.py", line 6 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "/code/api/models/__init__.py", line 1 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "<frozen importlib._bootstrap>", line 1204 in _gcd_import
  File "/usr/lib/python3.11/importlib/__init__.py", line 126 in import_module
  File "/usr/local/lib/python3.11/dist-packages/django/apps/config.py", line 269 in import_models
  File "/usr/local/lib/python3.11/dist-packages/django/apps/registry.py", line 116 in populate
  File "/usr/local/lib/python3.11/dist-packages/django/__init__.py", line 24 in setup
  File "/usr/local/lib/python3.11/dist-packages/django/core/wsgi.py", line 12 in get_wsgi_application
  File "/code/librephotos/wsgi.py", line 11 in <module>
  File "<frozen importlib._bootstrap>", line 241 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 940 in exec_module
  File "<frozen importlib._bootstrap>", line 690 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1147 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1176 in _find_and_load
  File "<frozen importlib._bootstrap>", line 1204 in _gcd_import
  File "/usr/lib/python3.11/importlib/__init__.py", line 126 in import_module
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/util.py", line 371 in import_app
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 48 in load_wsgiapp
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 58 in load
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 67 in wsgi
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 146 in load_wsgi
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/base.py", line 134 in init_process
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/workers/ggevent.py", line 147 in init_process
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 609 in spawn_worker
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 642 in spawn_workers
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 571 in manage_workers
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/arbiter.py", line 211 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 72 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/base.py", line 236 in run
  File "/usr/local/lib/python3.11/dist-packages/gunicorn/app/wsgiapp.py", line 67 in run
  File "/usr/local/bin/gunicorn", line 8 in <module>
Extension modules: greenlet._greenlet, zope.interface._zope_interface_coptimizations, gevent.libev.corecext, gevent._gevent_c_greenlet_primitives, gevent._gevent_c_hub_local, gevent._gevent_c_waiter, gevent._gevent_c_hub_primitives, gevent._gevent_c_ident, gevent._gevent_cgreenlet, gevent._gevent_c_abstract_linkable, gevent._gevent_cevent, gevent._gevent_cqueue, gevent._gevent_c_semaphore, gevent._gevent_c_imap, gevent._gevent_clocal, psutil._psutil_linux, psutil._psutil_posix, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator, markupsafe._speedups, charset_normalizer.md (total: 32)```

Could you tell me if we should upgrade torch librairie to latest one ?

Thanks,
Regards

**Comment by derneuere** [comment]
I upgraded the torch libraries on the dev tag. Could you pull that and check, if that fixed the issue for you?

**Comment by fpoisson35** [comment]
Hello,
Issue is still the same :

```
root@c4bbc7c1a0c5:/code# pip show torch
DEPRECATION: Loading egg at /usr/local/lib/python3.11/dist-packages/dlib-19.24.99-py3.11-linux-x86_64.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330
Name: torch
Version: 2.1.2+cpu
Summary: Tensors and Dynamic neural networks in Python with strong GPU acceleration
Home-page: https://pytorch.org/
Author: PyTorch Team
Author-email: packages@pytorch.org
License: BSD-3
Location: /usr/local/lib/python3.11/dist-packages
Requires: filelock, fsspec, jinja2, networkx, sympy, typing-extensions
Required-by: sentence-transformers, timm, torchvision
root@c4bbc7c1a0c5:/code# python -c 'import torch'
Illegal instruction (core dumped)
root@c4bbc7c1a0c5:/code# 
```

**Comment by fpoisson35** [comment]
i will try to make some tests with this library, it seems to fail with its internal tests tools too :

```
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin# echo $LD_LIBRARY_PATH
:/usr/local/lib/python3.11/dist-packages/torch/lib
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin# ./test_api 
Illegal instruction (core dumped)
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin# ./test_jit 
Illegal instruction (core dumped)
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin# ls
FileStoreTest  HashStoreTest  TCPStoreTest  core  protoc  protoc-3.13.0.0  test_api  test_cpp_rpc  test_dist_autograd  test_edge_op_registration  test_jit  test_lazy  test_tensorexpr  torch_shm_manager  tutorial_tensorexpr
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin# file core
core: ELF 64-bit LSB core file, x86-64, version 1 (SYSV), SVR4-style, from './test_jit', real uid: 0, effective uid: 0, real gid: 0, effective gid: 0, execfn: './test_jit', platform: 'x86_64'
root@c4bbc7c1a0c5:/usr/local/lib/python3.11/dist-packages/torch/bin#
```

**Comment by derneuere** [comment]
Can you provide more information about your system? What CPU do you use? Do you use a virtualization system like proxmox etc.?

**Comment by fpoisson35** [comment]
My hardware server is under Debian 11.8 and here is my /proc/cpuinfo:

```
cat /proc/cpuinfo 
processor	: 0
vendor_id	: AuthenticAMD
cpu family	: 16
model		: 6
model name	: AMD Turion(tm) II Neo N54L Dual-Core Processor
stepping	: 3
microcode	: 0x10000c8
cpu MHz		: 800.000
cache size	: 1024 KB
physical id	: 0
siblings	: 2
core id		: 0
cpu cores	: 2
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 5
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16 popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr hw_pstate vmmcall npt lbrv svm_lock nrip_save
bugs		: tlb_mmatch apic_c1e fxsave_leak sysret_ss_attrs null_seg amd_e400 spectre_v1 spectre_v2
bogomips	: 4392.56
TLB size	: 1024 4K pages
clflush size	: 64
cache_alignment	: 64
address sizes	: 48 bits physical, 48 bits virtual
power management: ts ttp tm stc 100mhzsteps hwpstate

processor	: 1
vendor_id	: AuthenticAMD
cpu family	: 16
model		: 6
model name	: AMD Turion(tm) II Neo N54L Dual-Core Processor
stepping	: 3
microcode	: 0x10000c8
cpu MHz		: 2200.000
cache size	: 1024 KB
physical id	: 0
siblings	: 2
core id		: 1
cpu cores	: 2
apicid		: 1
initial apicid	: 1
fpu		: yes
fpu_exception	: yes
cpuid level	: 5
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16 popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr hw_pstate vmmcall npt lbrv svm_lock nrip_save
bugs		: tlb_mmatch apic_c1e fxsave_leak sysret_ss_attrs null_seg amd_e400 spectre_v1 spectre_v2
bogomips	: 4392.56
TLB size	: 1024 4K pages
clflush size	: 64
cache_alignment	: 64
address sizes	: 48 bits physical, 48 bits virtual
power management: ts ttp tm stc 100mhzsteps hwpstate
```

I'm running a portainer docker instance for starting librephotos docker compose as a stack :
```
# docker ps
CONTAINER ID   IMAGE                                      COMMAND                  CREATED        STATUS                 PORTS                                                      NAMES
0916020d134b   reallibrephotos/librephotos-proxy:dev      "/docker-entrypoint.…"   4 hours ago    Up 4 hours             0.0.0.0:3000->80/tcp                                       proxy
c4bbc7c1a0c5   reallibrephotos/librephotos:dev            "/entrypoint.sh"         4 hours ago    Up 4 hours             8001/tcp                                                   backend
5670cc378527   reallibrephotos/librephotos-frontend:dev   "/serve"                 4 hours ago    Up 4 hours             3000/tcp, 8080/tcp                                         frontend
68f5323db02a   postgres:13                                "docker-entrypoint.s…"   4 hours ago    Up 4 hours (healthy)   5432/tcp                                                   db
74def90ef0ea   portainer/portainer-ce                     "/portainer"             3 months ago   Up 4 hours             0.0.0.0:8000->8000/tcp, 9443/tcp, 0.0.0.0:9009->9000/tcp   portainer
```

**Comment by fpoisson35** [comment]
Hello,

I build python 3.11.7 on my server and install latest torch release 2.1.2 and i have the same behaviour :

```
# /usr/local/bin/python3.11 -v -c "import torch"
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'posix' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# /usr/local/lib/python3.11/encodings/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/encodings/__init__.py
# code object from '/usr/local/lib/python3.11/encodings/__pycache__/__init__.cpython-311.pyc'
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <class '_frozen_importlib.FrozenImporter'>
# /usr/local/lib/python3.11/encodings/__pycache__/aliases.cpython-311.pyc matches /usr/local/lib/python3.11/encodings/aliases.py
# code object from '/usr/local/lib/python3.11/encodings/__pycache__/aliases.cpython-311.pyc'
import 'encodings.aliases' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc515490>
import 'encodings' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc5018d0>
# /usr/local/lib/python3.11/encodings/__pycache__/utf_8.cpython-311.pyc matches /usr/local/lib/python3.11/encodings/utf_8.py
# code object from '/usr/local/lib/python3.11/encodings/__pycache__/utf_8.cpython-311.pyc'
import 'encodings.utf_8' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc503290>
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <class '_frozen_importlib.FrozenImporter'>
import 'io' # <class '_frozen_importlib.FrozenImporter'>
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <class '_frozen_importlib.FrozenImporter'>
import '_collections_abc' # <class '_frozen_importlib.FrozenImporter'>
import 'genericpath' # <class '_frozen_importlib.FrozenImporter'>
import 'posixpath' # <class '_frozen_importlib.FrozenImporter'>
import 'os' # <class '_frozen_importlib.FrozenImporter'>
import '_sitebuiltins' # <class '_frozen_importlib.FrozenImporter'>
Processing user site-packages
Processing global site-packages
Adding directory: '/usr/local/lib/python3.11/site-packages'
Processing .pth file: '/usr/local/lib/python3.11/site-packages/distutils-precedence.pth'
# /usr/local/lib/python3.11/site-packages/_distutils_hack/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/_distutils_hack/__init__.py
# code object from '/usr/local/lib/python3.11/site-packages/_distutils_hack/__pycache__/__init__.cpython-311.pyc'
import '_distutils_hack' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc2e8450>
import 'site' # <class '_frozen_importlib.FrozenImporter'>
Python 3.11.7 (main, Dec 30 2023, 12:49:54) [GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
# /usr/local/lib/python3.11/site-packages/torch/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/torch/__init__.py
# code object from '/usr/local/lib/python3.11/site-packages/torch/__pycache__/__init__.cpython-311.pyc'
# extension module 'math' loaded from '/usr/local/lib/python3.11/lib-dynload/math.cpython-311-x86_64-linux-gnu.so'
# extension module 'math' executed from '/usr/local/lib/python3.11/lib-dynload/math.cpython-311-x86_64-linux-gnu.so'
import 'math' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc2f2750>
# /usr/local/lib/python3.11/__pycache__/platform.cpython-311.pyc matches /usr/local/lib/python3.11/platform.py
# code object from '/usr/local/lib/python3.11/__pycache__/platform.cpython-311.pyc'
# /usr/local/lib/python3.11/collections/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/collections/__init__.py
# code object from '/usr/local/lib/python3.11/collections/__pycache__/__init__.cpython-311.pyc'
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/local/lib/python3.11/__pycache__/keyword.cpython-311.pyc matches /usr/local/lib/python3.11/keyword.py
# code object from '/usr/local/lib/python3.11/__pycache__/keyword.cpython-311.pyc'
import 'keyword' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc32ee90>
# /usr/local/lib/python3.11/__pycache__/operator.cpython-311.pyc matches /usr/local/lib/python3.11/operator.py
# code object from '/usr/local/lib/python3.11/__pycache__/operator.cpython-311.pyc'
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc32f8d0>
# /usr/local/lib/python3.11/__pycache__/reprlib.cpython-311.pyc matches /usr/local/lib/python3.11/reprlib.py
# code object from '/usr/local/lib/python3.11/__pycache__/reprlib.cpython-311.pyc'
import 'reprlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc340510>
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc30f810>
# /usr/local/lib/python3.11/re/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/re/__init__.py
# code object from '/usr/local/lib/python3.11/re/__pycache__/__init__.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/enum.cpython-311.pyc matches /usr/local/lib/python3.11/enum.py
# code object from '/usr/local/lib/python3.11/__pycache__/enum.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/types.cpython-311.pyc matches /usr/local/lib/python3.11/types.py
# code object from '/usr/local/lib/python3.11/__pycache__/types.cpython-311.pyc'
import 'types' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc353990>
# /usr/local/lib/python3.11/__pycache__/functools.cpython-311.pyc matches /usr/local/lib/python3.11/functools.py
# code object from '/usr/local/lib/python3.11/__pycache__/functools.cpython-311.pyc'
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc361ed0>
import 'enum' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc352ad0>
# /usr/local/lib/python3.11/re/__pycache__/_compiler.cpython-311.pyc matches /usr/local/lib/python3.11/re/_compiler.py
# code object from '/usr/local/lib/python3.11/re/__pycache__/_compiler.cpython-311.pyc'
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/local/lib/python3.11/re/__pycache__/_parser.cpython-311.pyc matches /usr/local/lib/python3.11/re/_parser.py
# code object from '/usr/local/lib/python3.11/re/__pycache__/_parser.cpython-311.pyc'
# /usr/local/lib/python3.11/re/__pycache__/_constants.cpython-311.pyc matches /usr/local/lib/python3.11/re/_constants.py
# code object from '/usr/local/lib/python3.11/re/__pycache__/_constants.cpython-311.pyc'
import 're._constants' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc39bfd0>
import 're._parser' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc399410>
# /usr/local/lib/python3.11/re/__pycache__/_casefix.cpython-311.pyc matches /usr/local/lib/python3.11/re/_casefix.py
# code object from '/usr/local/lib/python3.11/re/__pycache__/_casefix.cpython-311.pyc'
import 're._casefix' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc39f910>
import 're._compiler' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc38e590>
# /usr/local/lib/python3.11/__pycache__/copyreg.cpython-311.pyc matches /usr/local/lib/python3.11/copyreg.py
# code object from '/usr/local/lib/python3.11/__pycache__/copyreg.cpython-311.pyc'
import 'copyreg' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc3a8bd0>
import 're' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc3516d0>
import 'platform' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc306d10>
# /usr/local/lib/python3.11/__pycache__/textwrap.cpython-311.pyc matches /usr/local/lib/python3.11/textwrap.py
# code object from '/usr/local/lib/python3.11/__pycache__/textwrap.cpython-311.pyc'
import 'textwrap' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc30f410>
# /usr/local/lib/python3.11/ctypes/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/ctypes/__init__.py
# code object from '/usr/local/lib/python3.11/ctypes/__pycache__/__init__.cpython-311.pyc'
# extension module '_ctypes' loaded from '/usr/local/lib/python3.11/lib-dynload/_ctypes.cpython-311-x86_64-linux-gnu.so'
# extension module '_ctypes' executed from '/usr/local/lib/python3.11/lib-dynload/_ctypes.cpython-311-x86_64-linux-gnu.so'
import '_ctypes' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc3b32d0>
# /usr/local/lib/python3.11/__pycache__/struct.cpython-311.pyc matches /usr/local/lib/python3.11/struct.py
# code object from '/usr/local/lib/python3.11/__pycache__/struct.cpython-311.pyc'
# extension module '_struct' loaded from '/usr/local/lib/python3.11/lib-dynload/_struct.cpython-311-x86_64-linux-gnu.so'
# extension module '_struct' executed from '/usr/local/lib/python3.11/lib-dynload/_struct.cpython-311-x86_64-linux-gnu.so'
import '_struct' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc3c6490>
import 'struct' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc3c6090>
# /usr/local/lib/python3.11/ctypes/__pycache__/_endian.cpython-311.pyc matches /usr/local/lib/python3.11/ctypes/_endian.py
# code object from '/usr/local/lib/python3.11/ctypes/__pycache__/_endian.cpython-311.pyc'
import 'ctypes._endian' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc1a0490>
import 'ctypes' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc3ab990>
# /usr/local/lib/python3.11/__pycache__/inspect.cpython-311.pyc matches /usr/local/lib/python3.11/inspect.py
# code object from '/usr/local/lib/python3.11/__pycache__/inspect.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/ast.cpython-311.pyc matches /usr/local/lib/python3.11/ast.py
# code object from '/usr/local/lib/python3.11/__pycache__/ast.cpython-311.pyc'
import '_ast' # <class '_frozen_importlib.BuiltinImporter'>
# /usr/local/lib/python3.11/__pycache__/contextlib.cpython-311.pyc matches /usr/local/lib/python3.11/contextlib.py
# code object from '/usr/local/lib/python3.11/__pycache__/contextlib.cpython-311.pyc'
import 'contextlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc1d1610>
import 'ast' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc1b6b50>
# /usr/local/lib/python3.11/__pycache__/dis.cpython-311.pyc matches /usr/local/lib/python3.11/dis.py
# code object from '/usr/local/lib/python3.11/__pycache__/dis.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/opcode.cpython-311.pyc matches /usr/local/lib/python3.11/opcode.py
# code object from '/usr/local/lib/python3.11/__pycache__/opcode.cpython-311.pyc'
# extension module '_opcode' loaded from '/usr/local/lib/python3.11/lib-dynload/_opcode.cpython-311-x86_64-linux-gnu.so'
# extension module '_opcode' executed from '/usr/local/lib/python3.11/lib-dynload/_opcode.cpython-311-x86_64-linux-gnu.so'
import '_opcode' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc236890>
import 'opcode' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc223cd0>
import 'dis' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc1d1f10>
# /usr/local/lib/python3.11/collections/__pycache__/abc.cpython-311.pyc matches /usr/local/lib/python3.11/collections/abc.py
# code object from '/usr/local/lib/python3.11/collections/__pycache__/abc.cpython-311.pyc'
import 'collections.abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc244ad0>
# /usr/local/lib/python3.11/importlib/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/importlib/__init__.py
# code object from '/usr/local/lib/python3.11/importlib/__pycache__/__init__.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/warnings.cpython-311.pyc matches /usr/local/lib/python3.11/warnings.py
# code object from '/usr/local/lib/python3.11/__pycache__/warnings.cpython-311.pyc'
import 'warnings' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc244550>
import 'importlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc244950>
import 'importlib.machinery' # <class '_frozen_importlib.FrozenImporter'>
# /usr/local/lib/python3.11/__pycache__/linecache.cpython-311.pyc matches /usr/local/lib/python3.11/linecache.py
# code object from '/usr/local/lib/python3.11/__pycache__/linecache.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/tokenize.cpython-311.pyc matches /usr/local/lib/python3.11/tokenize.py
# code object from '/usr/local/lib/python3.11/__pycache__/tokenize.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/token.cpython-311.pyc matches /usr/local/lib/python3.11/token.py
# code object from '/usr/local/lib/python3.11/__pycache__/token.cpython-311.pyc'
import 'token' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc267150>
import 'tokenize' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc247210>
import 'linecache' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc4bf910>
import 'inspect' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc3b2150>
# /usr/local/lib/python3.11/site-packages/torch/__pycache__/_utils.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/torch/_utils.py
# code object from '/usr/local/lib/python3.11/site-packages/torch/__pycache__/_utils.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/traceback.cpython-311.pyc matches /usr/local/lib/python3.11/traceback.py
# code object from '/usr/local/lib/python3.11/__pycache__/traceback.cpython-311.pyc'
import 'traceback' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc282bd0>
# /usr/local/lib/python3.11/__pycache__/typing.cpython-311.pyc matches /usr/local/lib/python3.11/typing.py
# code object from '/usr/local/lib/python3.11/__pycache__/typing.cpython-311.pyc'
# extension module '_typing' loaded from '/usr/local/lib/python3.11/lib-dynload/_typing.cpython-311-x86_64-linux-gnu.so'
# extension module '_typing' executed from '/usr/local/lib/python3.11/lib-dynload/_typing.cpython-311-x86_64-linux-gnu.so'
import '_typing' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc2926d0>
import 'typing' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc28f0d0>
import 'torch._utils' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc273ed0>
# /usr/local/lib/python3.11/site-packages/torch/__pycache__/_utils_internal.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/torch/_utils_internal.py
# code object from '/usr/local/lib/python3.11/site-packages/torch/__pycache__/_utils_internal.cpython-311.pyc'
# /usr/local/lib/python3.11/logging/__pycache__/__init__.cpython-311.pyc matches /usr/local/lib/python3.11/logging/__init__.py
# code object from '/usr/local/lib/python3.11/logging/__pycache__/__init__.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/weakref.cpython-311.pyc matches /usr/local/lib/python3.11/weakref.py
# code object from '/usr/local/lib/python3.11/__pycache__/weakref.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/_weakrefset.cpython-311.pyc matches /usr/local/lib/python3.11/_weakrefset.py
# code object from '/usr/local/lib/python3.11/__pycache__/_weakrefset.cpython-311.pyc'
import '_weakrefset' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc0b6a50>
import 'weakref' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc09e5d0>
# /usr/local/lib/python3.11/__pycache__/string.cpython-311.pyc matches /usr/local/lib/python3.11/string.py
# code object from '/usr/local/lib/python3.11/__pycache__/string.cpython-311.pyc'
import '_string' # <class '_frozen_importlib.BuiltinImporter'>
import 'string' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc0d0c90>
# /usr/local/lib/python3.11/__pycache__/threading.cpython-311.pyc matches /usr/local/lib/python3.11/threading.py
# code object from '/usr/local/lib/python3.11/__pycache__/threading.cpython-311.pyc'
import 'threading' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc0d1d10>
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>
import 'logging' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc08dc90>
# /usr/local/lib/python3.11/__pycache__/tempfile.cpython-311.pyc matches /usr/local/lib/python3.11/tempfile.py
# code object from '/usr/local/lib/python3.11/__pycache__/tempfile.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/shutil.cpython-311.pyc matches /usr/local/lib/python3.11/shutil.py
# code object from '/usr/local/lib/python3.11/__pycache__/shutil.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/fnmatch.cpython-311.pyc matches /usr/local/lib/python3.11/fnmatch.py
# code object from '/usr/local/lib/python3.11/__pycache__/fnmatch.cpython-311.pyc'
import 'fnmatch' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc10ba90>
import 'errno' # <class '_frozen_importlib.BuiltinImporter'>
# extension module 'zlib' loaded from '/usr/local/lib/python3.11/lib-dynload/zlib.cpython-311-x86_64-linux-gnu.so'
# extension module 'zlib' executed from '/usr/local/lib/python3.11/lib-dynload/zlib.cpython-311-x86_64-linux-gnu.so'
import 'zlib' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc11ecd0>
# /usr/local/lib/python3.11/__pycache__/bz2.cpython-311.pyc matches /usr/local/lib/python3.11/bz2.py
# code object from '/usr/local/lib/python3.11/__pycache__/bz2.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/_compression.cpython-311.pyc matches /usr/local/lib/python3.11/_compression.py
# code object from '/usr/local/lib/python3.11/__pycache__/_compression.cpython-311.pyc'
import '_compression' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc128b50>
# destroy bz2
# /usr/local/lib/python3.11/__pycache__/lzma.cpython-311.pyc matches /usr/local/lib/python3.11/lzma.py
# code object from '/usr/local/lib/python3.11/__pycache__/lzma.cpython-311.pyc'
# destroy lzma
import 'shutil' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc108190>
# /usr/local/lib/python3.11/__pycache__/random.cpython-311.pyc matches /usr/local/lib/python3.11/random.py
# code object from '/usr/local/lib/python3.11/__pycache__/random.cpython-311.pyc'
# /usr/local/lib/python3.11/__pycache__/bisect.cpython-311.pyc matches /usr/local/lib/python3.11/bisect.py
# code object from '/usr/local/lib/python3.11/__pycache__/bisect.cpython-311.pyc'
# extension module '_bisect' loaded from '/usr/local/lib/python3.11/lib-dynload/_bisect.cpython-311-x86_64-linux-gnu.so'
# extension module '_bisect' executed from '/usr/local/lib/python3.11/lib-dynload/_bisect.cpython-311-x86_64-linux-gnu.so'
import '_bisect' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc12be10>
import 'bisect' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc12b910>
# extension module '_random' loaded from '/usr/local/lib/python3.11/lib-dynload/_random.cpython-311-x86_64-linux-gnu.so'
# extension module '_random' executed from '/usr/local/lib/python3.11/lib-dynload/_random.cpython-311-x86_64-linux-gnu.so'
import '_random' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc12bc90>
# extension module '_sha512' loaded from '/usr/local/lib/python3.11/lib-dynload/_sha512.cpython-311-x86_64-linux-gnu.so'
# extension module '_sha512' executed from '/usr/local/lib/python3.11/lib-dynload/_sha512.cpython-311-x86_64-linux-gnu.so'
import '_sha512' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7f82dc12bf90>
import 'random' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc108710>
import 'tempfile' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc0b4a50>
import 'torch._utils_internal' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc08d650>
# /usr/local/lib/python3.11/site-packages/torch/__pycache__/torch_version.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/torch/torch_version.py
# code object from '/usr/local/lib/python3.11/site-packages/torch/__pycache__/torch_version.cpython-311.pyc'
# /usr/local/lib/python3.11/site-packages/torch/__pycache__/version.cpython-311.pyc matches /usr/local/lib/python3.11/site-packages/torch/version.py
# code object from '/usr/local/lib/python3.11/site-packages/torch/__pycache__/version.cpython-311.pyc'
import 'torch.version' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dbf05f50>
import 'torch.torch_version' # <_frozen_importlib_external.SourceFileLoader object at 0x7f82dc08d5d0>
Instruction non permise
``` 

I will try to investigate more on my server...

Thanks

**Comment by fpoisson35** [comment]
I found that the issue is from the torch release 2.1.x, while it works correctly  from your docker backend image when i downgrade to release 2.0.1 with this command :
```
pip3 install torch==2.0.1 torchvision==0.15.2 torchaudio==2.0.2 --break-system-packages --index-url https://download.pytorch.org/whl/cpu
```
as mentioned by the PyTorch website https://pytorch.org/get-started/previous-versions/

Now with this image i can run the torch verification code:

```
# docker run -it reallibrephotos/librephotos:perso
root@0881381af458:/code# pip3 show torch
DEPRECATION: Loading egg at /usr/local/lib/python3.11/dist-packages/dlib-19.24.99-py3.11-linux-x86_64.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330
Name: torch
Version: 2.0.1+cpu
Summary: Tensors and Dynamic neural networks in Python with strong GPU acceleration
Home-page: https://pytorch.org/
Author: PyTorch Team
Author-email: packages@pytorch.org
License: BSD-3
Location: /usr/local/lib/python3.11/dist-packages
Requires: filelock, jinja2, networkx, sympy, typing-extensions
Required-by: sentence-transformers, timm, torchaudio, torchvision
root@0881381af458:/code# 
root@0881381af458:/code# python3
Python 3.11.4 (main, Jun  9 2023, 07:59:55) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> x = torch.rand(5, 3)
>>> print(x)
tensor([[0.8398, 0.9389, 0.5779],
        [0.1924, 0.9218, 0.0519],
        [0.8832, 0.1766, 0.4320],
        [0.5181, 0.4207, 0.8300],
        [0.5241, 0.8038, 0.7997]])
>>> 
```

So i don't think there is any issue with my docker release or configuration but with PyTorch release, in  my case i use docker-ce from debian:

```
# dpkg -l docker-ce
Souhait=inconnU/Installé/suppRimé/Purgé/H=à garder
| État=Non/Installé/fichier-Config/dépaqUeté/échec-conFig/H=semi-installé/W=attend-traitement-déclenchements
|/ Err?=(aucune)/besoin Réinstallation (État,Err: majuscule=mauvais)
||/ Nom            Version                       Architecture Description
+++-==============-=============================-============-====================================================
ii  docker-ce      5:24.0.7-1~debian.11~bullseye amd64        Docker: the open-source application container engine
# docker --version
Docker version 24.0.7, build afdd53b
```

**Comment by fpoisson35** [comment]
Hello and happy new year !

For your information i opened a new issue on PyTorch  after some tests with your docker file :
[116623](https://github.com/pytorch/pytorch/issues/116623)

**Comment by derneuere** [comment]
Thanks for looking into it and reporting it upstream! Can you revert to an old version of LibrePhotos, while PyTorch finds and fixes the bug, or do you need a revert to the previous version?

**Comment by fpoisson35** [comment]
No i update the backend /entrypoint.sh with a pip3 command for installing release 2.0.1 before running the first python command.

**Comment by fpoisson35** [comment]
Hello,
The pytorch issue is now closed, and it works with nighlty build, i suppose that next release 2.3.0 will correct this issue.
Thanks,
Regards,

**Comment by derneuere** [comment]
This is fixed in latest now :)
